### PR TITLE
[PF-663] update jenkins to latest LTS

### PR DESCRIPTION
**Pull request by IppX**
# Motivation

Update to latest LTS to fix security issues.
Install pipeline-github plugin to enable feeback in PR features.

# Description
- update to latest Jenkins LTS
- update plugins accordingly
- add pipeline-github plugin
- fix docker-compose

### CODE REVIEW

File: plugins.txt [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-github-branch-source:2.9.2
+blueocean:1.24.5
+configuration-as-code:1.47
+docker-workflow:1.26
+github-branch-source:2.10.2
 greenballs:1.15.1
 job-dsl:1.77
-kubernetes:1.25.7
-parameterized-trigger:2.39
+kubernetes:1.29.2
```
Comment (renanqts) [inline plugins.txt:10]: Careful here, that was the plugin that breaks before

**Review by renanqts — APPROVED** [review]
LGTM

File: plugins.txt (hunk 1 above)
Comment (IppX) [reply plugins.txt:10]: Has critical security issues, I hope the double output is not there anymore but I think it overrules cosmetics anyways.

Comment (IppX) [reply plugins.txt:10]: Seems like it's fixed:
https://jenkins.oss-eks.dev.emnify.io/job/test-debug-ansible-pipeline/4/console
VS
https://jenkins.oss-eks.dev.emnify.io/job/test-debug-ansible-pipeline/5/console

**Pull request merged**